test/spoom/cli/coverage_test.rb

```diff
@@ -489,6 +489,17 @@ def test_finish_on_original_branch
         assert_equal("fake-branch", @project.git_current_branch)
       end
 
+      def test_error_raised_when_metrics_fails
```

Comment:
Is this test still relevant?

Reply:
Nope, just cleaning that up as we speak ;p